.github/actions/build-sign-publish-chainlink/action.yml

```diff
@@ -93,12 +93,12 @@ runs:
           SHARED_BUILD_ARGS=$(cat << EOF
         COMMIT_SHA=${GIT_COMMIT_SHA}
         CL_CHAIN_DEFAULTS=/ccip-config
+        CL_SOLANA_CMD=
```

Comment:
if it is empty and the default arg is in Dockerfile why not remove it?

Reply:
In the case of CCIP, they actually do not want the default behavior (for now). There was some logic in the GoReleaser config generator to remove it but GoReleaser was removed and the standard Dockerfile sets it as the default. (GoReleaser was using a separate Dockerfile.)